src/element-service.js

```diff
@@ -26,29 +27,6 @@ import {
 } from './service';
 import {userAssert} from './log';
 
-/**
- * Returns a promise for a service for the given id and window. Also expects an
- * element that has the actual implementation. The promise resolves when the
- * implementation loaded. Users should typically wrap this as a special purpose
- * function (e.g. Services.viewportForDoc(...)) for type safety and because the
- * factory should not be passed around.
- * @param {!Window} win
- * @param {string} id of the service.
- * @param {string} extension Name of the custom extension that provides the
- *     implementation of this service.
- * @param {boolean=} opt_element Whether this service is provided by an element,
- *     not the extension.
- * @return {!Promise<*>}
- */
-export function getElementService(win, id, extension, opt_element) {
```

Comment:
This function wasn't used anywhere?

Reply:
This function is used in the `extensions-impl`. I moved it from there and it's only used there. However, I felt like (a) `extensions-impl` is already big enough, and (b) I really wanted all "script" stuff to be in one place. Not a lot of tests affected because its a very simple move.

Reply:
Also some tests are still failing and I'm fixing them.